lms/models/lti_registration.py

```diff
@@ -33,3 +33,7 @@ class LTIRegistration(CreatedUpdatedMixin, BASE):
     """Location of the public keys provied by the issuer for this registration"""
     token_url = sa.Column(sa.UnicodeText, nullable=False)
     """Endpoint to request new oauth tokens to use with LTIA APIs"""
+
+    application_instances = sa.orm.relationship(
```

Comment:
The other side of of `back_populate` in application_instance.

Useful avoid writing a ORM query on the admin pages.